src/data/ACTIONS/root/PLD.ts

```diff
@@ -125,48 +123,74 @@ export const PLD = ensureActions({
 		icon: iconUrl(2519),
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
-		breaksCombo: false,
+		statusesApplied: [
+			'SUPPLICATION_READY',
+		],
 	},
 	CONFITEOR: {
 		id: 16459,
 		name: 'Confiteor',
 		icon: iconUrl(2518),
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
-		combo: {
-			start: true,
-		},
+		// Technically incorrect, but this way we don't get spurious broken combos
+		// combo: {
+		// 	start: true,
+		// },
```

Comment:
The Confiteor combo is a combo that get's broken by PLD's Fast Blade, Riot Blade, and Royal Authority. I think we still need to classify it as a combo. The Atonement follow up abilities do not combo nor break combos, so if you put that in Atonement and it's follow up abilities, things will work as intended. @czyba

Reply:
Unfortunately, I don't think it'll work. When I mark Atonement, Supplication and Sepulchre as `breakCombo: false` and make the Confiteor Combo an actual combo again, I end up with this: 

![image](https://github.com/user-attachments/assets/cc6bf084-3d74-443c-aeb9-93651f122994)

This is an issue I had during development, and I don't quite understand why it doesn't occur normally. Probably some difference between development and production build?

Anyways when I circumvent that error with:

```diff
diff --git a/src/parser/core/modules/Combos.tsx b/src/parser/core/modules/Combos.tsx
index 1978221c..b6f3c5e8 100644
--- a/src/parser/core/modules/Combos.tsx
+++ b/src/parser/core/modules/Combos.tsx
@@ -326,7 +326,7 @@ export class Combos extends Analyser {
                        <Table.Body>
                                {
                                        data.map(issue => {
-                                               const startEvent = issue.context[0]
+                                               const startEvent = issue.context[0] ?? {}
                                                const brokenTime = issue.type !== 'timeout' ? issue.breaker.timestamp : startEvent.timestamp + COMBO_TIMEOUT

                                                return <Table.Row key={startEvent.timestamp}>


```

I get the following view, with the changes you mentioned:

![image](https://github.com/user-attachments/assets/3116b694-8857-4b42-8452-7e7543a8b7df)

The Confiteor Combo breaks the Fast Blade combo. And after the combo the Royal Authority is marked as uncomboed.

A different approach would be to implement

```ts
	/**
	 * Jobs MAY override this to indicate whether this broken combo is allowed.
	 * Return true to indicate an allowable combo break, which will result in this break not being recorded and the current combo will be cleared with no other side effects.
	 * Return false to record the combo break and display it to the user
	 * @param event The event that caused the combo break
	 * @param context The active combo chain that is being broken
	 * @returns true if this combo break should not be recorded
	 */
	protected isAllowableComboBreak(_breaker: ComboBreak, _context: Array<Events['damage']>): boolean {
		return false
	}
```

for the PLD Combos. I orignally did that for the Confiteor combo, but that didn't solve the problem that the Combo Analyzer still marked the Royal Authority / Riot Blade as uncomboed, since we essentially ended the previous combo.

I also tried to Mark the Confiteor Combo Skills as not breaking combo to no avail.

If you've got an idea why this occurs, I'd be glad to fix it. I couldn't figure it out without delving deep into the Core Combos Analyzer.

Reply:
@Styrfire I think I figured it out. The EW Branch also marks the combo as uncomboed! We just missed it because it was added as a layer.

https://github.com/xivanalysis/xivanalysis/blob/a5371bb565b001cfa091f935beadcbf5518eab0a/src/data/ACTIONS/layers/patch6.3.ts#L28-L39

So essentially the layer did the same thing I did within this PR directly.